install/0000_90_machine-config-operator_01_prometheus-rules.yaml

```diff
@@ -43,3 +43,12 @@ spec:
             severity: warning
           annotations:
             message: "Kubelet health failure threshold reached"
+    - name: system-memory-exceeds-reservation
+      rules:
+        - alert: SystemMemoryExceedsReservation
+          expr: |
+            sum by (node) (container_memory_rss{id="/system.slice"}) > ((sum by (node) (kube_node_status_capacity{resource="memory"} - kube_node_status_allocatable{resource="memory"})) * 0.9)
+          labels:
+            severity: warning
+          annotations:
+            message: "System memory usage of {{ $value | humanize }} on {{ $labels.node }} exceeds 90% of the reservation"
```

Comment:
Messages should be acitonable.  So you need to recommend next steps.  I would suggest playing around with something like the following additions:

> Reserved memory ensures system processes can function even when the node is fully allocated and protects against workload out of memory events impacting the proper functioning of the node. The reservation may be increased (link to docs) when running nodes with high numbers of pods.

Or similar

Reply:
@smarterclayton better now?